src/snyk/snykCode/codeSettings.ts

```diff
@@ -81,6 +80,8 @@ export class CodeSettings implements ICodeSettings {
         // eslint-disable-next-line no-await-in-loop
         settings = await this.getSastSettings();
         if (settings?.sastEnabled && !settings?.localCodeEngine.enabled) {
+          // eslint-disable-next-line no-await-in-loop
+          await this.updateIsCodeEnabled();
```

Comment:
Is this part necessary? We call `updateIsCodeEnabled()` in the caller fn in `snykLib.ts:64`